.puppet-lint.rc

```diff
@@ -0,0 +1,55 @@
+# Use this file to instruct puppet-lint to ignore certain checks.
```

Comment:
can you please remove the file? We manage all the puppet-lint settings in our `Rakefile`.